src/smartpeak/source/core/RawDataProcessor.cpp

```diff
@@ -1994,9 +2109,30 @@ namespace SmartPeak
     LOGD << "END ClearData";
   }
 
+  ParameterSet CalculateMDVs::getParameterSchema() const
+  {
+    std::map<std::string, std::vector<std::map<std::string, std::string>>> param_struct({
+    {"CalculateMDVs", {
+      {
+        {"name", "mass_intensity_type"},
+        {"type", "string"},
+        {"value", "norm_sum"},
+        {"description", ""},
+        {"valid_strings", "['norm_sum','norm_max']"}
+      },
+      {
+        {"name", "feature_name"},
+        {"type", "string"},
+        {"value", "intensity"},
+        {"description", ""},
```

Comment:
We can use the following for the description:

"The name of the FeatureMap attribute to use.  Examples include `peak_apex_int`, `peak_area`, and `intensity`."